src/app/pages/gdpr/gdpr-data-request/gdpr-data-request.component.html

```diff
@@ -0,0 +1,27 @@
+<ng-container *ngIf="!!(error$ | async) === true">
+  <ish-error-message [error]="error$ | async" [toast]="false"></ish-error-message>
+  <h1 data-testing-id="empty-confirmation-title">
+    {{ 'gdpr.request.personal.data.request.title' | translate }}
+  </h1>
+  <p [innerHTML]="'gdpr.request.personal.data.confirmation.link.general.questions.details' | translate"></p>
```

Comment:
`[innerHTML]` won't work with links of type `<a href=\"route://contact\">` but if one would use it with `<a href=\"/contact\">` the PWA would not behave like a single page application.
So for this we have the `[ishServerHtml]` directive that should be used instead of `[innerHTML]`.